index.js

```diff
@@ -1,39 +1,40 @@
 'use strict'
 
 const fp = require('fastify-plugin')
-const elasticsearch = require('elasticsearch')
-
-function fastifyElasticSearch (fastify, options, next) {
-  const client = options.client || new elasticsearch.Client(options)
-  const requestTimeout = options.timeout
-
-  client.ping(
-    {
-      // ping usually has a 3000ms timeout
-      requestTimeout
-    },
-    function (err) {
-      if (err) {
-        next(err)
-      } else {
-        fastify.log.debug('elasticsearch cluster is available')
-        // plugin is ready
-        next()
-      }
+const { Client } = require('@elastic/elasticsearch')
+
+async function fastifyElasticsearch (fastify, options) {
```

Comment:
There is no need to use a default value, if the user doesn't pass an options object, the client **will** throw, because the `node` configuration is required.